main.py

```diff
@@ -51,13 +55,19 @@ def reAward(fname):
 ##(/5) Task 3: CREATE code that will open a file and extract all the phone numbers 
 
 def rePhone(fname):
-  #open file
-  #loop through the file
-  #extract the specific phone numbers regex
+  #open file X
+  #loop through the file X
+  #extract the specific phone numbers regex X
   #if the length of the extraction is not empty
   #print the phone number
-  pass
+  fhand= open(fname, "r")
+
+  for line in fhand:
+    extr = re.findall("^..:\s(\S*)", line)
```

Comment:
Look at the phone number pattern ###-###-####
extr = re.findall('\d{3}-\d{3}-\d{4}', line)